general/app/development/development-guide.md

```diff
@@ -415,5 +415,5 @@ You can learn more about this in the [Acceptance testing for the Moodle App](./t
 
 - [Moodle App Scripts: gulp push](./scripts/gulp-push.md)
 - [Moodle App Accessibility](../accessibility.md)
-- [Moodle App Deep Linking](./deep-linking.md)
+- [Moodle App Deep Linking](./links/deep-linking.md)
```

Comment:
```suggestion
- [Moodle App Link Handling](./link-handling)
```
I guess we should link to the entire section now. If nothing else, I think the new documentation is more relevant than deep linking so if we only link to one, maybe I'd prefer this one.

Reply:
Using the section gave me a broken link error, so now I'm linking to the first page.